Tasks/winget/task.yaml

```diff
@@ -1,19 +1,31 @@
+# This is a winget package installation task for Dev Box.
+
 name: winget
+description: Installs a package using winget
+author: Microsoft Corporation
 command: './main.ps1 -ConfigurationFile {{configurationFile}} -DownloadUrl {{downloadUrl}} -Package {{package}} -RunAsUser {{runAsUser}}'
 inputs:
   configurationFile:
     defaultValue: ''
     type: 'string'
     required: false
+    description: |
+      The path to the winget configuration file.
+      For example, "winget.json".
+      Visit https://docs.microsoft.com/windows/package-manager/winget/ to learn
+      more about winget.
   downloadUrl:
     defaultValue: ''
     type: 'string'
     required: false
+    description: The URL to download the package from.
```

Comment:
Should read `The URL to download the winget config yaml file from. The file will be placed under so and so path.` but with better grammar and filling in the blanks; ask Dayton about this or try to reverse engineer it from the ps1 file.